core-graphics/src/event.rs

```diff
@@ -505,7 +505,6 @@ impl<'tap_life> CGEventTap<'tap_life> {
                     callback_ref: Box::from_raw(cbr),
                 })
             } else {
-                Box::from_raw(cbr);
```

Comment:
This is not useless; it prevents memory being leaked. We could assign it to _ if that makes the warning go away.

Reply:
Yes, it'll make it go way. I'll change it.